src/DisplayOperations.cpp

```diff
@@ -0,0 +1,42 @@
+#include <iostream>
+#include <fstream>
+#include <exception>
+#include "DisplayOperations.h"
+using namespace std;
+
+/** Displays all the student entries */
+void DisplayAllOperation::DisplayAll()
+{
+    int size=vect.size();
+    if(0==size)
+    {
+        cout << "No Student Record Present" << endl;
+    }
+    else
+    {
+        cout << "\nId\tName\tBranch\tLocation" << endl;
+        vector<mystruct>::iterator ptr;
+        ptr=vect.begin();
+        mystruct mptr;
+        for (ptr = vect.begin(); ptr < vect.end(); ptr++)
```

Comment:
you can use range based loops ref: https://www.geeksforgeeks.org/range-based-loop-c/